File: go-web-basics/code-samples/hello-world-server/hello-world.go
package main

import (
    "fmt"
    "net/http"
)

func serveHello(w http.ResponseWriter, r *http.Request) {
    fmt.Fprintf(w, "Hello world!")
}

func main() {
    http.HandleFunc("/", serveHello)
    http.ListenAndServe(":1123", nil)
}

File: go-web-basics/code-samples/hello-world-server/hello-world.js
var http = require('http')

function serveHello(req, res){
  var hello = 'Hello world!'
  res.writeHead(200, {
    'Content-Length' : hello.length,
    'Content-Type'   : 'text/plain'
  });
  res.end(hello, 'UTF-8')
}

http.createServer(serveHello).listen(1123)
